JS/Lesson24/goksel/style.css

```diff
@@ -0,0 +1,142 @@
+* {
+    box-sizing: border-box;
+    font-family: sans-serif;
+}
+
+body {
+    background-color: #000;
+}
+
+.form-container {
+    background-color: #fff;
+    border-radius: 9px;
+    width: 900px;
+    margin: 75px auto;
+}
+
+/* .form is for to remove content */
+.form {
```

Comment:
If you have a class that only hides an element, name the class 'hidden'. This way you would be able to say exactly what happens if the class is applied to an element and you don't need to write comments as above with explanations.